composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
+        , "symfony/dependency-injection": "^2.8|^3.0"
     }
     , "require-dev": {
           "matthiasnoback/symfony-dependency-injection-test": "^0.7"
+        , "phpunit/phpunit": "^4.8|^5.0"
```

Comment:
👎 PHPunit relies on Symfony's Yaml component and can thus can create not workable dependencies during testing. However, as PHPunit isn't used at runtime, this bundle can still work against these Symfony versions.

As a real usage: PHPunit didn't support Symfony 3 close to when it was released. Having this dependency here means this bundle couldn't be tested against Symfony 3 until just before the release.